### PR TITLE
perf: optimize pfd and simconnect requests

### PR DESCRIPTION
**Pull request by Saschl**
Fixes #[issue_no]

## Summary of Changes

 - Removes usages of SIMCONNECT_PERIOD_ONCE as it's not recommended for regularly requesting data (https://docs.flightsimulator.com/html/Programming_Tools/SimConnect/API_Reference/Structures_And_Enumerations/SIMCONNECT_PERIOD.htm?rhhlterm=SIMCONNECT_PERIOD_ONCE&rhsearch=SIMCONNECT_PERIOD_ONCE)
 - PFD drawing optimizations
   - Reduce amount of unnecessary draws
   - Use translate3d for translation to make use of GPU acceleration

## Additional context

Discord username (if different from GitHub):

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: fbw-a32nx/src/systems/instruments/src/PFD/SpeedIndicator.tsx [hunk 1]
```diff
@@ -991,11 +992,84 @@ class SpeedTarget extends DisplayComponent <{ bus: ArincEventBus }> {
...
+            <g id="SpeedMargins">
+                <path
+                    id="UpperSpeedMargin"
+                    class="Fill Magenta"
+                    d="m19.7 80.5 h 5.3577 v 0.7 h-5.3577 z"
+                    visibility={this.upperSpeedMarginVisibility}
+                    transform={this.upperMarginTransform}
+                />
+                <path
+                    id="UpperSpeedMargin"
```
Comment (tracernz) [inline fbw-a32nx/src/systems/instruments/src/PFD/SpeedIndicator.tsx]:
```suggestion
                    id="LowerSpeedMargin"
```

File: fbw-a32nx/src/systems/instruments/src/PFD/AttitudeIndicatorHorizon.tsx [hunk 2]
```diff
@@ -55,7 +55,7 @@ class HeadingBug extends DisplayComponent<{ bus: ArincEventBus, isCaptainSide: b
         });
 
         sub.on('heading').handle((h) => {
-            this.heading = h;
+            this.heading = h.value;
```
Comment (tracernz) [inline fbw-a32nx/src/systems/instruments/src/PFD/AttitudeIndicatorHorizon.tsx:58]: Seems a backwards step to discard the SSM, and in this case it should check the SSM.

**Comment by Saschl** [comment]
I see there's a lot more to optimize/cleanup, however that would be suitable for another PR. Main goal here was to fix the FMA borders being constantly changed and to get rid of SIMCONNECT_PERIOD_ONCE

File: fbw-a32nx/src/systems/instruments/src/PFD/AttitudeIndicatorHorizon.tsx [hunk 3]
```diff
@@ -26,67 +27,55 @@ const ValueSpacing = 10;
...
+    private horizonHeadingBug = FSComponent.createRef<SVGGElement>();
 
+    private readonly visibilitySub = MappedSubject.create(([heading, attitude, fdActive, selectedHeading]) => {
+        const headingDelta = getSmallestAngle(selectedHeading, heading.value);
         const offset = headingDelta * DistanceSpacing / ValueSpacing;
+        const inRange = Math.abs(offset) <= DisplayRange + 10;
+        if (!fdActive && attitude.isNormalOperation() && heading.isNormalOperation() && inRange) {
+            return true;
+        }
+        return false;
```
Comment (BlueberryKing) [inline fbw-a32nx/src/systems/instruments/src/PFD/AttitudeIndicatorHorizon.tsx:47]:
```suggestion
        return !fdActive && attitude.isNormalOperation() && heading.isNormalOperation() && inRange
```

**Review by BlueberryKing — CHANGES_REQUESTED** [review]

**Review by BlueberryKing — APPROVED** [review]
Nice work!

**Comment by auroraisluna** [comment]
QA Report

Discord: alepouna
Object of testing: #7835 
Tier of Testing : 1
Date : 23/7/2023

Testing Process:
- Spawn in airport
- Observe PFD operational 
- Observe aircraft performance

Testing Results:
Passed

Notes: 
My CPU loves you Saschl. 

**FPS comparison**
Latest dev branch (at the time of writting) 
image

PR
image

:stonks:

**Pull request merged**